### ISSUE TITLE
WPF ChakraCore missing package

### ISSUE THREAD

**Issue by kyrre** [Microsoft/react-native-windows#1154]
The ChakraCore version used in the WPF project is no longer available.

Severity	Code	Description	Project	File	Line	Suppression State
Error		NuGet Package restore failed for project Native: Unable to find version '1.4.1-preview-00010-42060' of package 'Microsoft.ChakraCore'.
  C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\: Package 'Microsoft.ChakraCore.1.4.1-preview-00010-42060' is not found on source 'C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\'.
  https://www.myget.org/F/chakracore-preview/api/v3/index.json: Package 'Microsoft.ChakraCore.1.4.1-preview-00010-42060' is not found on source 'https://www.myget.org/F/chakracore-preview/api/v3/index.json'.
  https://api.nuget.org/v3/index.json: Package 'Microsoft.ChakraCore.1.4.1-preview-00010-42060' is not found on source 'https://api.nuget.org/v3/index.json'.
  https://www.nuget.org/packages/Microsoft.ChakraCore: The V2 feed at 'https://www.nuget.org/packages/Microsoft.ChakraCore/FindPackagesById()?id='Microsoft.ChakraCore'' returned an unexpected status code '404 Not Found'.

Version: react-native-windows@0.45.0-rc.1

**Comment by rozele** [comment]
@matthargett any thoughts on this?

**Comment by matthargett** [comment]
crap, it looks like the ChakraCore team removed the preview package. We should update this to latest 1.5.x release and backport to 42, 43, 44.

**Comment by berstend** [comment]
I'm running into the same issue when trying to bootstrap a new wpf project. :-)

```
react-native init foowpf
cd foowpf
yarn add -D rnpm-plugin-wpf
yarn add -D rnpm-plugin-windows
react-native wpf
react-native run-wpf
```

`Failed to restore the NuGet packages`

Any way to hotfix this?

**Comment by berstend** [comment]
I naively attempted to replace the 2 occurrences of `Microsoft.ChakraCore.1.4.1-preview-00010-42060` with `Microsoft.ChakraCore.1.4.4` (in `packages.config` and `.csproj`) and now the demo compiles correctly.

**Comment by matthargett** [comment]
can you submit a pull request that updates those package versions in the project templates and CLI to latest ChakraCore 1.5.2 release?

**Comment by sm2017** [comment]
![problem](https://user-images.githubusercontent.com/19644997/27825588-473f97be-60c5-11e7-81ad-575fb15d024d.png)

any solution ?

**Comment by kyrre** [comment]
@sm2017  just manually change the dependency from Microsoft.ChakraCore.1.4.1-preview-00010-42060 to Microsoft.ChakraCore.1.4.4

**Comment by sm2017** [comment]
@kyrre 
in packages file 

` <package id="Microsoft.ChakraCore" version="1.4.1-preview-00010-42060" targetFramework="net46" developmentDependency="true" />`

Change to

`<package id="Microsoft.ChakraCore" version="1.4.4" targetFramework="net46" developmentDependency="true" />`

its your mean ?

**Comment by sm2017** [comment]
Ok
Now Show This Error 

> Build failed with message Error: Command failed: "C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe" "C:/Users/PC/Desktop/AwesomeProject/wpf/AwesomeProject.sln" /clp:NoSummary;NoItemAndPropertyList;Verbosity=minimal /nologo /p:Configuration=Debug /p:Platform=x86 /p:AppxBundle=Never. Check your build configuration.

![problem](https://user-images.githubusercontent.com/19644997/27826820-d2c15468-60c9-11e7-8472-063ab817bbc8.png)

**Comment by sm2017** [comment]
?????

**Comment by matthargett** [comment]
sorry for the delay. #1185 fixes this issue by using official ChakraCore 1.5.2 release from nuget.org. Once it's merged to master, we'll cherry pick into 42-46 branches and re-release those. /cc @digitalinfinity

**Comment by sm2017** [comment]
Any update?

**Comment by berstend** [comment]
@sm2017 did you also update the ChakraCore version to 1.4.4 in the `.csproj` file as well?

**Comment by rozele** [comment]
Closed by #1185
